src/launchUnelevated.js

```diff
@@ -0,0 +1,28 @@
+var objShell = new ActiveXObject("shell.application");

```

Comment:
I'm just going to trust that this is all correct :)

Reply:
Maybe use `let` instead of var.

Reply:
Yes, and I tested it as much as I can. It works.

Unfortunately the Windows Script Host (cscript.exe) is nowhere near any modern ECMAScript standard. `let` is not supported.